server.js

```diff
@@ -14,14 +14,17 @@ const app = express();
 
 const PORT = 3002;
 
+//require weather placeholder json today, will hit live weather API's json tomorrow
+const weatherData = require('./data/weather.json');
+
```

Comment:
Make sure to add CORS!

Reply:
also you should add

require('dotenv').config();

const PORT = process.env.PORT || 3002;

Reply:
Hi there, Shane left some great feedback here! :) 💯 

- After you load the module for CORS: `const cors = require('cors');`, we will need to use the middleware `app.use(cors());` 👍